Lib/test/test_json/test_speedups.py

```diff
@@ -36,6 +37,26 @@ def test_make_encoder(self):
             b"\xCD\x7D\x3D\x4E\x12\x4C\xF9\x79\xD7\x52\xBA\x82\xF2\x27\x4A\x7D\xA0\xCA\x75",
             None)
 
+    @test.support.cpython_only
```

Comment:
I think this test should be passed on other implementations that have compatible `c_make_encoder`.

Reply:
i thought about using `test.support.get_attribute()` to check whether `c_make_encoder()` is available, but `test_make_encoder()` just uses it, and would have failed if it hadn't existed.

maybe we can assume that every other C implementations implements `test_make_encoder()`?

Reply:
I think that if other C implementation doesn't implement `test_make_encoder()` we should get a request for making this test optional.